opentelemetry-sdk/src/opentelemetry/sdk/metrics/__init__.py

```diff
@@ -103,27 +146,36 @@ def __init__(
         description: str,
         unit: str,
         value_type: Type[metrics_api.ValueT],
+        meter: "Meter",
         label_keys: Sequence[str] = (),
         enabled: bool = True,
         monotonic: bool = False,
+        absolute: bool = True,
     ):
         self.name = name
         self.description = description
         self.unit = unit
         self.value_type = value_type
+        self.meter = meter
         self.label_keys = label_keys
         self.enabled = enabled
         self.monotonic = monotonic
+        self.absolute = absolute
         self.handles = {}
 
     def get_handle(self, label_set: LabelSet) -> BaseHandle:
         """See `opentelemetry.metrics.Metric.get_handle`."""
         handle = self.handles.get(label_set)
         if not handle:
             handle = self.HANDLE_TYPE(
-                self.value_type, self.enabled, self.monotonic
+                self.value_type,
+                self.enabled,
+                self.monotonic,
+                self.absolute,
+                # Aggregator will be created based off type of metric
+                self.meter.batcher.aggregator_for(self.__class__),
             )
-        self.handles[label_set] = handle
+            self.handles[label_set] = handle
```

Comment:
I see that handles are never released. Is it out of scope of this PR?

Reply:
Yes. I'm tracking this in [#399]